tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -228,25 +234,31 @@ def view_caller(self, request):
 
 
 class TestCanvasFileBasicLTILaunch:
-    def test_it_configures_frontend(self, context, canvas_request):
-        canvas_file_basic_lti_launch_caller(context, canvas_request)
+    def test_it_configures_frontend(self, context, pyramid_request):
+        context.is_canvas = True
```

Comment:
As this is the `TestCanvasFile...` I wonder if we should override the context fixture for this class to add this in.

Reply:
Replaced this with a class-level `@pytest.mark.usefixtures("is_canvas")`